pkg/common/cns-lib/vsphere/virtualcenter.go

```diff
@@ -385,6 +385,10 @@ func (vc *VirtualCenter) GetVsanDatastores(ctx context.Context) (map[string]*Dat
 		finder.SetDatacenter(dc.Datacenter)
 		datastoresList, err := finder.DatastoreList(ctx, "*")
 		if err != nil {
+			if _, ok := err.(*find.NotFoundError); ok {
+				log.Debugf("No datastores found on %q datacenter", dc.Name())
```

Comment:
IMO, this should be `log.Warnf`

Reply:
It is not mandated to have datastores in every DC right? Also we have debug logs for Not found errors in other parts of our code as well, so let us maintain consistency. For example: https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/csi/service/common/authmanager.go#L332

Reply:
debug should be fine.

Reply:
Let's say I have a VC configured with 2 DCs and one DC has no datastore. `Debugf` will mask such incorrect configuration being displayed in the logs. Init will be successful because of the `continue` at Line: 390 and we will never get a chance to indicate such incorrect configuration in the testbed. 

> For example: https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/csi/service/common/authmanager.go#L332
The example you are referring to is inside the method `IsFileServiceEnabled`. And this will be invoked for every file volume create call. May be that's why the log is `debugf`. However in this case(`GetVsanDatastores`) is called once during.

Reply:
We need to flag incorrect configurations which hinder our workflow. I do not think this is a harmful scenario to either the customer or to our controller. Why do you think having a DC with no datastore is incorrect?

Reply:
> Why do you think having a DC with no datastore is incorrect?
Because we will end up searching for objects in the inventory in such DCs which do not have datastores 

I am fine with `Debugf` for now.

Reply:
I just checked the entire workflow. The only info stored in `authmanager` is a map of `dsURL` to `govmomi DS object` and its corresponding `DC`. In cases where the DC is empty (no datastores), the DC will never be stored in this dsUrlInfoMap so we will not iterate over the objects in this DC if at all we use this map for such activities in future.